tests/dummy/app/states/todos.js

```diff
@@ -1,6 +1,8 @@
 import BufferedProxy from 'ember-buffered-proxy/proxy';
 
-BufferedProxy.reopenClass({
+const TodosProxy = BufferedProxy.extend();
+
+TodosProxy.reopenClass({
   initialState() {
     return {
       content: this.get('todo')
```

Comment:
Perhaps I was misunderstanding, but it seemed like if you reopened `BufferedProxy` and you had multiple state services the last one would win, clobbering the other `reopens` as each state service is initialized.